docker/runtime/bash/http-trigger/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:2.7-slim
+
+RUN pip install bottle==0.12.13 cherrypy==8.9.1 wsgi-request-logger prometheus_client
+
+ADD kubeless-shell.py /
+
+EXPOSE 8080
```

Comment:
I would inherit from `bitnami/minideb:jessie` image. This provides a minimal environment for debian and an useful command `install_packages` to install dependencies (https://github.com/bitnami/minideb).

Also I would add an ENTRYPOINT to install dependencies. For example:
```
ADD install_dependencies.sh /bin/

ENTRYPOINT /bin/install_dependencies.sh
```
being install_dependencies.sh:
```
#!/bin/bash

if [[ -f /tmp/requirements ]]; then
 install_packages -y $(cat /tmp/requirements)
fi
```

Reply:
Thanks for all the feedback! I just submitted the changes.